e2e/tests/functional/plugins/faultManagement/faultManagement.e2e.spec.js

```diff
@@ -0,0 +1,165 @@
+
+const { test, expect } = require('../../../../pluginFixtures');
+const path = require('path');
+const { first } = require('lodash');
+
+test.describe('The Fault Management Plugin', () => {
+    test.beforeEach(async ({ page }) => {
+        await startAndNavigateToFaultManagement(page);
+    });
+
+    test('Shows a criticality icon for every fault', async ({ page }) => {
+        const faultCount = await page.locator('c-fault-mgmt__list').count();
+        const criticalityIconCount = await page.locator('c-fault-mgmt__list-severity').count();
+
+        expect.soft(faultCount).toEqual(criticalityIconCount);
+    });
+
+    test('When selecting a fault, it has an "is-selected" class and it\'s information shows in the inspector', async ({ page }) => {
+        await selectFaultItem(page, 1);
+
+        const selectedFaultName = await page.locator('.c-fault-mgmt__list.is-selected .c-fault-mgmt__list-faultname').textContent();
+        const inspectorFaultNameCount = await page.locator(`.c-inspector__properties >> :text("${selectedFaultName}")`).count();
+
+        await expect.soft(page.locator('.c-faults-list-view-item-body > .c-fault-mgmt__list').first()).toHaveClass(/is-selected/);
+        expect.soft(inspectorFaultNameCount).toEqual(1);
+    });
+
+    test('When selecting multiple faults, no specific fault information is shown in the inspector', async ({ page }) => {
+        await selectFaultItem(page, 1);
+        await selectFaultItem(page, 2);
+
+        const selectedRows = page.locator('.c-fault-mgmt__list.is-selected .c-fault-mgmt__list-faultname');
+        expect.soft(await selectedRows.count()).toEqual(2);
+
+        const firstSelectedFaultName = await selectedRows.nth(0).textContent();
+        const secondSelectedFaultName = await selectedRows.nth(1).textContent();
+        const firstNameInInspectorCount = await page.locator(`.c-inspector__properties >> :text("${firstSelectedFaultName}")`).count();
+        const secondNameInInspectorCount = await page.locator(`.c-inspector__properties >> :text("${secondSelectedFaultName}")`).count();
+
+        expect.soft(firstNameInInspectorCount).toEqual(0);
+        expect.soft(secondNameInInspectorCount).toEqual(0);
+    });
+
+    test('Allows you to shelve a fault', async ({ page }) => {
+        const shelvedFaultName = await getFaultName(page, 2);
+
+        await shelveFault(page, 2);
+        await changeViewTo(page, 'shelved');
+
+        const shelvedViewFaultName = await getFaultName(page, 1);
+
+        expect.soft(shelvedFaultName).toEqual(shelvedViewFaultName);
+    });
+
+    test('Allows you to acknowledge a fault', async ({ page }) => {
+        const acknowledgedFaultName = await getFaultName(page, 3);
+
+        await acknowledgeFault(page, 3);
+        await changeViewTo(page, 'acknowledged');
+
+        const acknowledgedViewFaultName = await getFaultName(page, 1);
+
+        expect.soft(acknowledgedFaultName).toEqual(acknowledgedViewFaultName);
```

Comment:
what is the expected state after this has been acknowledged?

Reply:
That is shows up in the acknowledged view, which I'm testing here and it also get's an `is-acknowledged` class, I'll add a check for that.